README.md

```diff
@@ -78,8 +78,8 @@ module.exports = {
   ...
   module: {
     loaders: [
-      { test: /\.scss$/i, loader: extractCSS.extract(['css','sass']) },
-      { test: /\.less$/i, loader: extractLESS.extract(['css','less']) },
+      { test: /\.scss$/i, loader: extractCSS.extract(['css-loader','sass-loader']) },
```

Comment:
Related: https://github.com/webpack/extract-text-webpack-plugin/issues/265

The problem is that, if you use `use:`, you get a crash.